src/client/post/Write/editorReducer.js

```diff
@@ -72,7 +72,6 @@ const editor = (state = defaultState, action) => {
       return {
         ...state,
         error: null,
-        loading: false,
```

Comment:
don't we need loading flag?

Reply:
we redirect to another page with a timeout, and if the load value is set to false upon successful completion, the bootloader stops and after a 3-second delay page redirect